client/src/components/plugins/icons.js

```diff
@@ -0,0 +1,93 @@
+import { library } from "@fortawesome/fontawesome-svg-core";
```

Comment:
😍😍😍 . This would save me so much copy-pasted code.

Reply:
I'm glad you like it. But it shouldn't even be in the javascript at all. Adding a SVG background is well within the capabilities of the css layer.